tox.ini

```diff
@@ -27,6 +27,11 @@ deps =
     ./psycopg[docs]
     ./psycopg_pool
 
+[testenv:codespell]
+commands = codespell -L "alot,ans,ba,fo,te" docs psycopg* tests
```

Comment:
I've read about "alot" elsewhere, but what are those strange other words?

Reply:
*alot* is a misspelling but I put there on purpose, just to have some fun during the long hours writing documentation... :smile: 

Others are variable names and the likes I guess. `ans` is the DNS answer, `ba` is for `bytearray`... The use is localised and obvious and I don't feel a compelling need to fix them just to make the tool happy. Other are small strings use in tests: they might be changed but again I feel no pressure about it.

One that was uglier was fixed in the first commit here, that `noe` which is better called `no_e`, a function that strips the prefix from postgres `E'string'` for the sake of tests.

This codespell tool is nice: it doesn't report errors on ay asdfasdf that it doesn't understand but still does a good job to tell apart misspelling of human words. I don't know its rate of false negatives but the false positives seem very few.

Reply:
I see, thanks for clarifying; makes sense.

I'm totally on board with codespell as well.